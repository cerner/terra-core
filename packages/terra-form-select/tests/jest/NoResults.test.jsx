import React from 'react';
/* eslint-disable-next-line import/no-extraneous-dependencies */
import { shallowWithIntl } from 'terra-enzyme-intl';
import NoResults from '../../src/shared/_NoResults';

describe('NoResults', () => {
  it('should render a default NoResults', () => {
    const wrapper = shallowWithIntl(<NoResults value="value" />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render NoResults with custom content', () => {
    const wrapper = shallowWithIntl(<NoResults value="value" noResultContent={<div>Custom</div>} />);
    expect(wrapper).toMatchSnapshot();
  });
});
