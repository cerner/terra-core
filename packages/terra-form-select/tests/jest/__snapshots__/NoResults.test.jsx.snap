// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults should render NoResults with custom content 1`] = `
<div
  className="no-results"
>
  <div>
    Custom
  </div>
</div>
`;

exports[`NoResults should render a default NoResults 1`] = `
<div
  className="no-results"
>
  Terra.form.select.noResults
</div>
`;
